### ISSUE TITLE
Package name and scope

### ISSUE THREAD

**Issue by christophsax** [srlanalytics/BDFM#10]
Now, since we cover other methods than Bayesian as well, why not advertise it a bit more broadly? 

E.g. call it dfm or dfmr, both names are available. The latter would have the advantage that it is easy to search for your package on the internet.

**Comment by srlanalytics** [comment]
Could do. I had focused on Bayesian estimation because:

(1) It is the hardest one to code up correctly
(2) It is typically the most powerful for forecasting applications due to the ability to include prior information
(3) There are other packages that do ML dynamic factor models using Watson and Engle's version of the EM algorithm (which vastly superior to numerical optimization of the likelihood function)... this is how we do it too for `method = "ML"`. There are no packages that I'm aware of (or can find easily on google) that do Bayesian estimation of factor models using Durban and Koopman's disturbance smoother. While most of the methodology is laid out in their 2012 book they offer no suggestions on identification to ensure you're drawing from the same distribution at each MCMC iteration. They simply write:

>Though Bayesian estimation by Gibbs sampling sounds straight forward it is not due to the fact that factors are not identified.

I had to work out a reliable methodology for that bit on my own. 

(4) Some people (like me) think that a Bayesian approach to statistics is the "right" way to view the world... the frequentest idea of "true" parameters is not very satisfying. 

BDFM is the jargon used in the literature for Bayesian dynamic factor models and is thus appealing as it means something on it's own. However, if you think a broader approach would be easier to sell that's probably a better way to go. Let me know what you think...

**Comment by christophsax** [comment]
I agree on your point about about the *B*. I still prefer lower case, though. *bdfm* is easier to type than *BDFM*.

**Comment by srlanalytics** [comment]
Yes, lower case is probably more in keeping with the standard for R packages anyhow... _bdfm_ sounds good to me.

**Comment by srlanalytics** [comment]
Yes, I think we should. Can I leave that to you or shall I make a start on it?

**Comment by christophsax** [comment]
I did a PR #26 with renaming that passes the checks. Remaining steps on your side:

- [ ] merge PR
- [ ] rename GitHub repo 
- [ ] adjust the path in in your Git Client
- [ ] point to new location in travis
- [ ] point to new location in netlify

**Comment by srlanalytics** [comment]
Not sure about updating travis to the new repository name. However, I don't thing repository names are case sensitive (since it's a url) so it shouldn't be an issue. If you have a better idea about this let me know.
